src/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,98 @@
+import { Link, NavLink } from "react-router-dom";
+
+const navbar = () => {
+  return (
+    <div className="navbar flex flex-row justify-between items-center bg-black w-screen h-24 px-24">
+      <div className="container_pages">
+        {/* logo & home link */}
+        <Link
+          to="/"
+          className="
+           text-white
+            cursor-pointer
+            text-2xl
+            -tracking-wider 
+            leading-7 
+            font-noto 
+            font-bold
+            mr-20"
+        >
+          Stellar
+        </Link>
+
+        {/* pages links */}
+        <NavLink
+          to="/planatrip"
+          className={({ isActive }) => {
+            return (
+              "text-white cursor-pointer text-xs leading-7 font-robo font-bold hover:text-red-700 transition-colors duration-200 delay-75 mr-10" +
+              (!isActive ? "" : " text-red-700")
```

Comment:
Conditionals, well done 👍 

This method inside `className` and the classes that are applied seem to be duplicated, would be great to extract this to a single method so only one update would be needed if anything needed to change.